hashin.py

```diff
@@ -175,17 +182,25 @@ def run_single_package(
             new_lines += ' \\'
         new_lines += '\n'
 
-    if verbose:
-        _verbose('Editing', file)
     with open(file) as f:
         requirements = f.read()
```

Comment:
Change this to `old_requirements = f.read()` and 2 lines down pass in `old_requirements` (instead of `requirements`) to the `amend_requirements_content` function. 
Then you don't need lines 195-196 since you'll already have a copy of the whole old file content.